cli/src/cmd/create.rs

```diff
@@ -57,7 +73,18 @@ impl Cmd for CreateArgs {
         // Add arguments to constructor
         let provider = Provider::<Http>::try_from(self.eth.rpc_url()?)?;
         let params = match abi.constructor {
-            Some(ref v) => self.parse_constructor_args(v)?,
+            Some(ref v) => {
+                let mut constructor_args = self.constructor_args.clone();
+                if let Some(constructor_args_path) = &self.constructor_args_path {
+                    if !std::path::Path::new(&constructor_args_path).exists() {
+                        return Err(eyre::eyre!("constructor args path not found"));
+                    }
+                    let file = fs::read_to_string(constructor_args_path)?;
+                    let args = file.split(" ").map(|s| s.to_string()).collect::<Vec<String>>();
+                    constructor_args = args;
+                }
```

Comment:
```suggestion
                let constructor_args = if let Some(ref constructor_args_path) = self.constructor_args_path {
                    if !std::path::Path::new(&constructor_args_path).exists() {
                        eyre::bail!("constructor args path not found");
                    }
                    let file = fs::read_to_string(constructor_args_path)?;
                    file.split(" ").map(|s| s.to_string()).collect::<Vec<String>>()
                } else {
                    self.constructor_args
                };
```

this is a bit better

Reply:
Ah good looks getting rid of the `mut`